### ISSUE TITLE
Does not compile

### ISSUE THREAD

**Issue by BrunoDutertre** [SRI-CSL/yices2_ocaml_bindings#1]
``opam install yices2_bindings`` fails with this error:

```
# File "src/yices2_ext_bindings.ml", line 22, characters 64-67:
# 22 |   | List l -> Format.fprintf fmt "@[<hv 1>(%a)@]" (List.pp ~sep:" " pp_sexp) l
#                                                                      ^^^
# Error: The function applied to this argument has type
#          ?pp_start:unit List.printer ->
#          ?pp_stop:unit List.printer ->
#          ?pp_sep:unit List.printer ->
#          Format.formatter -> 'weak2 List.t -> unit
# This argument cannot be applied with label ~sep
# Command exited with code 2.
```

I've tried different versions of ocaml and they all failed with this error. Google didn't find
documentation for ``List.printer``.

**Comment by disteph** [comment]
Ah, it's the List.pp from the [Containers library](https://github.com/c-cube/ocaml-containers), which contains lots of modules that improve on the standard library's. Containers's move from version 2.8 to 3.0 has a [breaking change affecting for instance the type of `List.pp`](https://github.com/c-cube/ocaml-containers/issues/295). I'll update my code to reflect the change of Containers' API, and impose that the dependency on Containers is >= 3.0.

**Comment by disteph** [comment]
Should be fixed by version 0.2 of the bindings (i.e., commit 10ab9bdfd6b7d347fb71e1fcdfb9f54c22629f1e).
I have made a [merge request to opam's official repository](https://github.com/ocaml/opam-repository/pull/17191), so that version 0.2 is accessible via opam. It will be when the merge request is accepted.

**Comment by BrunoDutertre** [comment]
Thanks
